src/main/java/net/openhft/chronicle/hash/impl/VanillaChronicleHash.java

```diff
@@ -658,7 +658,6 @@ protected void cleanupOnClose() {
         // Releases nothing after resources.releaseManually(), only removes the cleaner
         // from the internal linked list of all cleaners.
         cleaner.clean();
-        ChronicleHashCloseOnExitHook.remove(this);
```

Comment:
This line should be there if the shutdown hook is not skipped.

Reply:
Fixed